resources/views/usergroups/edit.blade.php

```diff
@@ -3,24 +3,17 @@
 
 @section('content')
 
-    @include('statamic::partials.breadcrumb', [
-        'url' => cp_route('user-groups.show', $group->handle()),
-        'title' => $group->title()
-    ])
-
-    <h1 class="mb-3">@yield('title')</h1>
-
-    <collection-edit-form>
-    </collection-edit-form>
-
-    <user-group-publish-form
-        action="{{ cp_route('user-groups.update', $group->handle()) }}"
+    <user-publish-form
```

Comment:
I don't think you meant to change to `user-publish-form`. You probably want to keep it as `user-group-publish-form`.